### DISCUSSION TITLE
NPC spawner tool for chairs

### DISCUSSION THREAD

**Discussion by Asnro** [tsunamayo/Starship-EVO#4764] [Suggestions]
I know proper NPC crew will be implemented in the future, but for now ships would feel more lived-in if there was a tool to spawn NPCs to sit on designated chairs. How I imagine the tool to work is you click on a chair with it, and get a menu to select or randomize the appearance of an NPC who will sit down on the chair, similar to the NPCs who sit on chairs of AI-controlled ships.

**Comment by Horkas** [comment]
As a super cheap and easy solution, he could do it how Empyrion does it, and just add static crew objects that already have seats built in. Basically, the crew is just a physical object with an idle animation that repeats, that you place like any normal block. Its not the greatest solution, but it works. And thats all thats important. https://i.ytimg.com/vi/UD46nvwzFQI/maxresdefault.jpg
